build/index.js

```diff
@@ -38,25 +38,9 @@ var path = require("path"),
     }
   };
 
-exports.onCreateProject = function (devkitAPI, app, config, cb) {
+  exports.onBeforeBuild = function (devkitAPI, app, config, cb) {
```

Comment:
@CodeCombiner , I think the onCreateProject might be needed because notification icons were being deleted. We had changed it in https://github.com/hashcube/onesignal/pull/34/files
@vivekb91 can confirm.